src/clients/QBittorrent.ts

```diff
@@ -329,38 +327,51 @@ export default class QBittorrent implements TorrentClient {
 				(await this.isSubfolderContentLayout(searchee))
 					? "Subfolder"
 					: "Original";
-
-			const skipRecheckTorrent =
-				decision === Decision.MATCH_PARTIAL ? skipRecheck : true;
-
 			const formData = new FormData();
 			formData.append("torrents", buffer, filename);
-			formData.append("tags", TORRENT_TAG);
-			formData.append("category", newCategoryName);
-
-			if (autoTMM) {
-				formData.append("autoTMM", "true");
-			} else {
-				formData.append("autoTMM", "false");
-				formData.append("savepath", save_path);
-			}
+			// searchee was linked
 			if (path) {
-				formData.append("contentLayout", "Original");
-				formData.append("skip_checking", skipRecheck.toString());
-				formData.append("paused", (!skipRecheck).toString());
+				//because it was a linked file, turn off autotmm
+				//and set a save path and datacat
+				formData.append(
+					"autoTMM",
+					flatLinking && searchee.infoHash ? autoTMM : "false",
+				);
+				formData.append("savepath", save_path);
+				formData.append("category", linkCategory);
+				// tag category
+				formData.append(
+					"tags",
+					searchee.infoHash
+						? `${TORRENT_TAG},${newCategoryName}`
+						: category,
+				);
 			} else {
-				formData.append("contentLayout", contentLayout);
-				formData.append("skip_checking", skipRecheckTorrent.toString());
-				formData.append("paused", (!skipRecheckTorrent).toString());
+				// torrent-backed
```

Comment:
```suggestion
				// perfect match
```

Reply:
@ShanaryS said that partial can directly point to the original data...so this isn't necessarily a perfect match in the sense that they are identical torrent -> torrent, but data -> torrent in a similar way as a data match would be minus the linking?

file names would be perfect matches, just wasn't trying to be ambiguous about how we refer to perfect matches in other areas.

Reply:
ill update the comment accordingly though